include/spdk/nvme.h

```diff
@@ -1141,6 +1142,36 @@ int spdk_nvme_ns_cmd_reservation_report(struct spdk_nvme_ns *ns,
 					void *payload, uint32_t len,
 					spdk_nvme_cmd_cb cb_fn, void *cb_arg);
 
+/**
+ * \brief Sends an abort command to the Nvme device for the request indicated
+ *  by cid and sqid
+ *
+ * \param ctrlr controller to the Nvme device
+ * \param cid identification number of the command to be aborted.
+ * \param sqid submission queue number of the command to be aborted
+ * \param cb_fn a function pointer to the abort call back function
+ * \param cb_arg user argument to the abort command callback function
+ *
+ * \return 0 if successfully submitted, Less than 0 if it fails
+ *
+ * Note: The user must ensure to decrement the controller member
```

Comment:
I'm not sure this Note makes sense; why is the user responsible for this?  We should be able to handle this count internally in the NVMe library.